src/lxc/storage/storage.c

```diff
@@ -337,6 +337,11 @@ struct lxc_storage *storage_copy(struct lxc_container *c, const char *cname,
 	struct rsync_data data = {0};
 	char cmd_output[MAXPATHLEN] = {0};
 
+	if (!src) {
+		ERROR("No rootfs specified");
```

Comment:
Under what conditions do we get to this case?

I'm wondering whether the ERROR message is warranted.  Is it indicative of a particular misconfiguration on the user's part, or a bug in our code, or is it just a case that sometimes happens, and we should just return NULL quietly?

Reply:
If you remove the `lxc.rootfs.path` entry from the containers config file you'd default on copy but you'd work correctly on e.g. execute. That's why I want to check.

Reply:
Sorry - what does "default on copy" mean?  is that autocorrect for segfault?

Reply:
Ups, yes "segfault".